src/components/Layout/LayoutHeader.tsx

```diff
@@ -77,7 +77,10 @@ export const LayoutHeader = ({
               indexName={process.env.ALGOLIA_INDEX_NAME || ALGOLIA_INDEX_NAME}
               apiKey={process.env.ALGOLIA_API_KEY || ALGOLIA_API_KEY}
               searchParameters={{
-                facetFilters: [`platform:${isGen1 ? 'gen1' : currentPlatform}`]
+                facetFilters: [
+                  `platform:${currentPlatform}`,
+                  `gen:${isGen1 ? 'gen1' : 'gen2'}`
+                ]
```

Comment:
In Algolia, we defined the `platform` and `gen` facet filters to help filter the pages. By passing these strings in through the search params as facet filters, it's as if the user is searching with these keywords too.